java/code/src/com/suse/manager/webui/services/impl/SaltService.java

```diff
@@ -320,8 +327,26 @@ private String callToString(AbstractCall<?> call) {
         return String.format("[%s.%s]", call.getModuleName(), call.getFunctionName());
     }
 
+    private static Map<String, Object> filterPayload(Map<String, Object> payload) {
+        var kwarg = payload.get("kwarg");
+        if (kwarg == null) {
+            return payload;
+        }
+        var kwargFiltered = kwarg instanceof Map ?
+                ((Map<?, ?>) kwarg).entrySet().stream()
+                .filter(e -> e.getValue() != null)
+                .collect(Collectors.toMap(
+                        Entry::getKey,
+                        e -> SENSITIVE_DATA_PATTERN.matcher(e.getValue().toString()).find() ?
+                                "HIDDEN" :
+                                e.getValue())) :
+                kwarg;
+        payload.put("kwarg", kwargFiltered);
+        return payload;
+    }
+
     private String runnerCallToString(RunnerCall<?> call) {
-        return String.format(PAYLOAD_CALL_TEMPLATE, call, call.getPayload());
+        return String.format(PAYLOAD_CALL_TEMPLATE, call.getModuleName(), filterPayload(call.getPayload()));
```

Comment:
We may refactor the filter to it's own class, but then there's more code to be rewritten

Reply:
fine for now